docs/migration/10.x-pictograms.md

```diff
@@ -0,0 +1,175 @@
+# Pictograms
```

Comment:
Would this be 11.x-pictograms.md? 👀

Reply:
yes updated to `11.x-pictograms.md`!

Reply:
@joshblack regarding a migration guide for Pictograms React - it looks like didn't do that for icons? 👀

Reply:
@andreancardona IIRC the icons migration guide was specifically for `carbon-icons`. I think `@carbon/icons-react` was published with v10 and so was using the new names by default 👍

Reply:
done!